src/main.js

```diff
@@ -161,6 +167,13 @@ function createPercyClient() {
   return new PercyClient({ token, apiUrl, clientInfo });
 }
 
+function isEnabled() {
+  const hasRequiredVars = Boolean(process.env.PERCY_TOKEN) && Boolean(process.env.PERCY_PROJECT);
+  const hasDisableVar = parseInt(process.env.PERCY_ENABLE) === 0;
+  const hasForceEnable = parseInt(process.env.PERCY_ENABLE) === 1;
```

Comment:
"force enable" isn't really something we support because it won't work without project or token available. We usually only support turning it off when PERCY_ENABLE=0 and otherwise ignore it, so I'd consider dropping this forceEnable flow.

Reply:
Sounds good to me. I didn't really like the option, but I added it when I was trying to get the tests working.

Switched it over to just use a dummy token as well.